include/openssl/obj_mac.h

```diff
@@ -5447,6 +5447,14 @@
 #define NID_aes_256_siv         1200
 
 #endif /* OPENSSL_OBJ_MAC_H */
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
```

Comment:
This seems an odd place to be adding a copyright - and it looks to me like this file already has a copyright at the beginning???

Reply:
Yes, this diff is in the fixup commit. It was done by `make update`, and it's omission caused the CI to fail. I don't know why it was inserted by `make update`. This PR was supposed to be copyrights-only via comments. I was trying to avoid any actual code changes...

Reply:
That's because the `make update` copies the obj_compat.h here verbatim including the copyright comment from the file.

Reply:
I figured it was something like that. And obj_compat.h was updated in this PR to include a missing copyright.

Reply:
Dealt with in the `unix-Makefile.tmpl` file